CMakeLists.txt

```diff
@@ -98,7 +109,7 @@ if (${HASCXX20} MATCHES "yes")
 elseif (${HASCXX17} MATCHES "yes")
     set(CMAKE_CXX_STANDARD 17)
 else ()
-    message(FATAL_ERROR "Minimum C++ standard version is 17,
+    message("Minimum C++ standard version is 17,
```

Comment:
Thanks for pointing out. I agree to force using c++17-compiled ROOT. I removed this error because on windows we cannot use `root-config` to collect `HASCXX17`, which makes cmake failed on this line. I have excluded this check for windows.